boot-mongodb-elasticsearch/src/test/java/com/example/mongoes/web/controller/RestaurantControllerIntTest.java

```diff
@@ -1,22 +1,55 @@
 package com.example.mongoes.web.controller;
 
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
 import com.example.mongoes.common.AbstractIntegrationTest;
 import com.example.mongoes.document.Address;
-import com.example.mongoes.web.model.GradesRequest;
-import com.example.mongoes.web.model.RestaurantRequest;
+import com.example.mongoes.model.request.GradesRequest;
+import com.example.mongoes.model.request.RestaurantRequest;
+import java.time.Duration;
 import java.time.LocalDateTime;
 import java.util.List;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
 import org.springframework.data.geo.Point;
+import org.springframework.http.HttpHeaders;
 import org.springframework.http.MediaType;
 
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
```

Comment:
@coderabbitai, can you please help me with running each test case independently instead of dependent on order of execution